ordered_model/models.py

```diff
@@ -217,7 +206,7 @@ class OrderedModel(OrderedModelBase):
     Provides an ``order`` field.
     """
 
-    order = models.PositiveIntegerField(editable=False, db_index=True)
+    order = models.PositiveIntegerField(_('order'), editable=False, db_index=True)
```

Comment:
This strikes me as odd - would changing my translations used by gettext break/reset object ordering, leading to a model change?

Reply:
Translations don't affect model values in any way, because they are not stored in database

Reply:
I didn't understand first part, what arg I should have used explicitly? Either way it should have been added so that users can translate this field properly

Reply:
I think @shuckc is suggesting that this should be `verbose_name=_(‘order’)`, that way a migration isn’t triggered if the translation is changed. Is that right, @shuckc? (I am not sure how this works, but it sounds like you do, so I will defer to you for the call here. :)

Reply:
Yes, `verbose_name=_(‘order’)` is an improvement, but I still find it very odd to see a translated field used in the model specification, due to it triggering pointless migrations for all existing users when this code lands, as well as potentially on future translation changes. i8s should be higher up the stack in presentation layer. This is a field we do not display in our own admins - so why do you need to localise the verbose name?

Reply:
preferred outcome: revert this line completely, it causes non-structual migrations to track a value we do not display in any of this projects code.

Reply:
@shuckc @bfirsh 
I know that this will cause migration, but I do not agree with you that this is wrong, if this project had this from the start there would not have been any questions because it is good practice to do so, here are examples https://github.com/jazzband/django-model-utils/blob/master/model_utils/models.py. I myself provide `order` field in admin because I need  to know actual numeric value. We can't assume to know all of the users' use cases, but at least we should follow good practices. If there is such a strong disagreement about this, I don't see a big problem with reverting this line. But would suggest leaving it, because open source project should be a good example how to write proper code.

Reply:
@shuckc @bfirsh bump

Reply:
Ah, I see. I misunderstood and thought the migration would actually cause a database change. If it's just a no-op migration, that doesn't seem like a big deal.

I don't know enough about translations to know if it's good practice to add it to field definitions. From the looks of django-model-utils, perhaps it is? If it is, then I'm for the change.